tools/astarte_dev_tool/test/astarte_dev_tool_test.exs

```diff
@@ -1,8 +1,41 @@
+#
+# This file is part of Astarte.
+#
+# Copyright 2024 SECO Mind Srl
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
 defmodule AstarteDevToolTest do
   use ExUnit.Case
-  doctest AstarteDevTool
+  alias AstarteDevTool.Utilities.Auth
+
+  # doctest AstarteDevTool
+  describe "unit test" do
+    test "key" do
+      assert {:ok, private} = Auth.pem_key()
+      assert {:ok, public} = Auth.pem_key(private)
+    end
+
+    test "pem_keys/0" do
```

Comment:
as above, i'll not test against another function, stick with the ones that already exists (above test is enough, just change the name)